### ISSUE TITLE
Suddenly can't upload to pomf.se

### ISSUE THREAD

**Issue by Atario** [ShareX/ShareX#449]
I get this stack trace:

Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.ReadInternal()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at ShareX.UploadersLib.FileUploaders.Pomf.Upload(Stream stream, String fileName)
   at ShareX.UploadTask.UploadFile(Stream stream, String fileName)
   at ShareX.UploadTask.DoUpload(Int32 retry)

**Comment by Jaex** [comment]
I can upload. If there way problem then I couldn't upload too.

**Comment by Atario** [comment]
I didn't just make that stack trace up, dude.

**Comment by Jaex** [comment]
I can't solve server side problems.

**Comment by Atario** [comment]
If it were a server-side problem, you'd be getting it too, wouldn't you?

**Comment by Jaex** [comment]
No they would be fixed it when I checked it.

**Comment by Atario** [comment]
What?

**Comment by Jaex** [comment]
No they would be fixed it when I checked it.

**Comment by Atario** [comment]
I don't know what sentence you're trying to write, but it still gets the same error for me right now.

**Comment by Jaex** [comment]
Is "Show response" exist in right click menu? ![](https://a.pomf.se/icpxyw.png)

**Comment by Atario** [comment]
No, my right-click menu is much shorter:

![screenshot](http://i.imgur.com/c9k4pDL.png)

**Comment by Jaex** [comment]
Well, I can't fix if there is problem because I can't reproduce it.
But if uploader not works suddenly then it means 90% server side problem because codes not changing magically when it was working fine before in your PC.

**Comment by Atario** [comment]
Figured it out.  My antivirus program, Bitdefender Free, was blocking the response from update.php.  I don't know why it blocked just that page and not the whole website.  Or any of it at all, for that matter.  The stupid thing also provides no way to disable that behavior.  So I dumped it and went to Panda Free instead.

Sorry for the false alarm, but I hope others who get this problem will see this on a Google search someday…
